src/__tests__/Avatar.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import {Avatar} from '..'
 import theme from '../theme'
-import {px, render, percent, behavesAsComponent, checkExports} from '../utils/testing'
+import {px, render, behavesAsComponent, checkExports} from '../utils/testing'
 import {render as HTMLRender, cleanup} from '@testing-library/react'
 import {axe, toHaveNoViolations} from 'jest-axe'
-import 'babel-polyfill'
 import {COMMON} from '../constants'
+import 'jest-styled-components'
```

Comment:
TODO: The `jest-styled-components` docs suggest this [usage](https://github.com/styled-components/jest-styled-components#usage) (i.e. explicitly importing it in every test file), but we ought to be able to import it once, in a shared/common file. We just need its `toHaveStyleRule` function type monkey-patch on `jest`.